.github/CODEOWNERS

```
@@ -19,6 +19,7 @@
 
 # Event Ingestion
 /src/sentry/attachments/           @getsentry/owners-ingest
+/src/sentry/api/endpoints/relay/** @getsentry/owners-ingest
```

btw does the double asterisk have any special meaning here? I think for owning the entire directory, a trailing slash would also do.

---

In github codeowners there is just the single star. Double star is a shell thing for include subdirectories.
```suggestion
/src/sentry/api/endpoints/relay/* @getsentry/owners-ingest
```

---

Do you have a source for this? According to [GitHub's docs](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners#codeowners-syntax), they follow the [gitignore pattern format](https://git-scm.com/docs/gitignore#_pattern_format), which cites:

> An asterisk "*" matches anything except a slash.

We also use it in other places:

https://github.com/getsentry/sentry/blob/ab50a1d2294c3ce34c6a79084607200eda11a220/.github/CODEOWNERS#L8

---

I'll remove the glob, since files and sub-directories are owned automatically by specifying a parent directory.